bitnami/kubeapps/templates/dashboard/deployment.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ template "kubeapps.dashboard.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: dashboard
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" . ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.dashboard.replicaCount }}
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: dashboard
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/dashboard/configmap.yaml") . | sha256sum }}
```

Comment:
Just out of curiosity: why other pods don't have this annotation? (newbie to this label indeed)

Reply:
It's assigned only to the ones that mounts a configmap or secret. It's a way to detect changes in the ConfigMap/Secret and update the pods.

Reply:
Woah! That's really useful indeed. Just yesterday we hit an issue caused by a pod not being upgraded after a cm change!

https://kubernetes.slack.com/archives/C9D3TSUG4/p1620343770056800?thread_ts=1620331053.044800&cid=C9D3TSUG4